cmd/tendermint/commands/rollback.go

```diff
@@ -22,22 +24,28 @@ restarting Tendermint the transactions in block n will be re-executed against th
 application.
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			height, hash, err := RollbackState(conf)
+			height, hash, err := RollbackState(conf, removeBlock)
```

Comment:
can we add a confirmation input step? like "Are you really sure you want to rollback?"